spec/models/ncr/work_order_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe Ncr::WorkOrder do
+  describe 'fields_for_display' do
+    it "shows BA61 fields" do
+      wo = Ncr::WorkOrder.new(
+        amount: 1000, expense_type: "BA61", vendor: "Some Vend", 
+        not_to_exceed: false, emergency: true, rwa_number: "RWWAAA #",
+        building_number: Ncr::WorkOrder::BUILDING_NUMBERS[0],
+        office: Ncr::WorkOrder::OFFICES[0])
```

Comment:
There's a factory, but I chose not to use it here since all of the fields are being tested. I can swap it out, though.